### ISSUE TITLE
Wrong province for Canada: "Recovered"

### ISSUE THREAD

**Issue by nuno-aj-aniceto** [CSSEGISandData/COVID-19#2156]
On the Cases (infections) there is a funny Province/State for Canada: 'Recovered'

238	Recovered	Canada

Kind regards
:)

**Comment by stefancaspersz** [comment]
yes very strange - should this data be in the recovered time series file under Canada?
